services/tone_analyzer/v3.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2013,2016 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+module.exports = function (RED) {
+  var cfenv = require('cfenv');
+  
+  var services = cfenv.getAppEnv().services,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['services' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2651367456/issues/source?bid=3225115&fileBranchId=3324712#l20)